tests/compile_fail/display/shared_format_positional_placeholders.stderr

```diff
@@ -0,0 +1,17 @@
+error: shared format string cannot contain positional placeholders, use named placeholders instead
+ --> tests/compile_fail/display/shared_format_positional_placeholders.rs:2:11
+  |
+2 | #[display("Stuff({})")]
+  |           ^^^^^^^^^^^
+
+error: shared format string cannot contain positional placeholders, use named placeholders instead
+ --> tests/compile_fail/display/shared_format_positional_placeholders.rs:8:11
+  |
+8 | #[display("Stuff({0})")]
+  |           ^^^^^^^^^^^^
+
+error: shared format string does not support positional placeholders, use named placeholders instead
```

Comment:
This seems to me a quite misleading error, because in tests we clearly have the valid case `#[display("Variant {_variant} {_0}")]` where the positional placeholder is used.

Reply:
That's not a positional placeholder. That's a named placeholder for a tuple field. But I do understand the confusion. If you have a suggestion for better wording of the error, I'm happy to change it.